### ISSUE TITLE
Add form field labels

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#3310]
Our accessibility audit flagged missing form field labels.

Our accessibility audit flagged missing names for select elements.

## `all-production`
```
• Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrappe
d around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #all-production > h3 > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `federal-production`
```
 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrappe
d around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #federal-production > section > h3 > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `revenue-trends`
```
 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrappe
d around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #revenue-trends > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `gdp`
```
  • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrappe
d around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #gdp > h3 > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `employment`
```
 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrappe
d around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #employment > section:nth-child(1) > h3 > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `self-employment`
```
 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrappe
d around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #self-employment > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

**Comment by mcharg** [comment]
Closing in favor of new accessibility audit.
